src/LineUI/LineSet.tsx

```diff
@@ -229,13 +229,15 @@ const LineSet : React.FC<ILineSetProps> = ({ getCTM, boundaries, unit, size, lin
       styling={styling}
       lineMoveCallback={lineDragUpdate}
       lineMoveStartCallback={lineDragStart}
+      showSmallDirectionMark={lineSetData.showSmallDirectionMark}
+      overrideShowMoveHandle={lineSetData.showMoveHandle}
     />
   );});
 
   return (
     <g>
       {lines}
-      {handles}
+      {(lineSetData.showPointHandle ?? true) && handles}
```

Comment:
this is breaking storybook, 

please use || instead of ??

lineSetData.showPointHandle || true

![Screen Shot 2022-07-07 at 21 55 08](https://user-images.githubusercontent.com/10409078/177781781-63a88d6e-e0ee-42fb-9bb8-093f8a0988e0.png)